### PR TITLE
fix broken styling on safari and apply truncated for super long string

### PR DESCRIPTION
**Pull request by reaksmey1**
1. Fix old styling which is not support on safari
2. Apply truncate in case on long string

### CODE REVIEW

File: angular/core/components/group_page/group_previous_proposals_card/group_previous_proposals_card.coffee [hunk 1]
```diff
@@ -16,3 +16,21 @@ angular.module('loomioApp').directive 'groupPreviousProposalsCard', ->
 
     $scope.canShowMore = ->
       $scope.group.closedMotionsCount > 3
+
+angular.module('loomioApp').filter 'cut', ->
```
Comment (gdpelican) [inline angular/core/components/group_page/group_previous_proposals_card/group_previous_proposals_card.coffee]: We have a filter for this called 'truncate', check out `core/filters/truncate_filter.coffee` :)

We use it in subgroups_card.haml

Comment (gdpelican) [inline angular/core/components/group_page/group_previous_proposals_card/group_previous_proposals_card.coffee]: @reaksmey1 Any update on this? Would be great to merge this in, with that minor change!

Comment (reaksmey1) [inline angular/core/components/group_page/group_previous_proposals_card/group_previous_proposals_card.coffee]: @gdpelican yes, the pull request is updated

**Comment by gdpelican** [comment]
Looks good to me, thanks, @reaksmey1 !

**Pull request merged**